<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneSettings</class>
 <widget class="QDialog" name="SceneSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxCamera">
     <property name="title">
      <string>Camera :</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="Rotation">
        <item>
         <widget class="QLabel" name="rotation_label">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Rotation</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Rotation_X">
          <item>
           <widget class="QLabel" name="rotation_x_label">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rotation_x_value">
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <double>-100000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Rotation_Y">
          <item>
           <widget class="QLabel" name="rotation_y_label">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rotation_y_value">
            <property name="minimum">
             <double>-100000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Rotation_Z">
          <item>
           <widget class="QLabel" name="rotation_z_label">
            <property name="text">
             <string>Z</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rotation_z_value">
            <property name="minimum">
             <double>-100000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Position">
        <item>
         <widget class="QLabel" name="position_label">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Position</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Position_X">
          <item>
           <widget class="QLabel" name="position_x_label">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="position_x_value">
            <property name="minimum">
             <double>-100000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Position_Y">
          <item>
           <widget class="QLabel" name="position_y_label">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="position_y_value">
            <property name="minimum">
             <double>-100000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Position_Z">
          <item>
           <widget class="QLabel" name="position_z_label">
            <property name="text">
             <string>Z</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="position_z_value">
            <property name="minimum">
             <double>-100000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="viewPlaneDistance">
        <item>
         <widget class="QLabel" name="viewPlaneDistance_label">
          <property name="text">
           <string>viewPlaneDistance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="viewPlaneDistance_value">
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="viewPlaneWidth">
        <item>
         <widget class="QLabel" name="viewPlaneWidth_label">
          <property name="text">
           <string>viewPlaneWidth</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="viewPlaneWidth_value">
          <property name="value">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="viewPlaneHeight">
        <item>
         <widget class="QLabel" name="viewPlaneHeight_label">
          <property name="text">
           <string>viewPlaneHeight</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="viewPlaneHeight_value">
          <property name="value">
           <double>0.350000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SceneSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SceneSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
